socorro/external/es/super_search_fields.py

```diff
@@ -1530,24 +1530,6 @@ def number_field(
         "query_type": "bool",
         "storage_mapping": {"type": "boolean"},
     },
-    "exploitability": {
-        "data_validation_type": "enum",
-        "description": "An automated estimate of how exploitable this crash is.",
-        "form_field_choices": ["high", "normal", "low", "none", "unknown", "error"],
```

Comment:
This suggests there were all these possible values, but since december 2021 it's only been "unknown".